soapui/src/main/xsd/soapui/soapui.xsd

```diff
@@ -1118,7 +1118,7 @@
             <xsd:element name="interface" type="xsd:string"></xsd:element>
             <xsd:element name="operation" type="xsd:string"></xsd:element>
             <xsd:element name="path" type="xsd:string"></xsd:element>
-            <xsd:element name="port" type="xsd:int"></xsd:element>
```

Comment:
We cannot change data format in the project. It hurts backward compatibility. I'll close the PR.